blueprints/codes/eurocode/nen_9997_1_c2_2017/chapter_2_basic_of_geotechnical_design/formula_2_4.py

```diff
@@ -0,0 +1,47 @@
+"""Formula 2.2 from NEN 9997-1+C2:2017: Chapter 2: Basis of geotechnical design."""
+
+from blueprints.codes.eurocode.nen_9997_1_c2_2017 import NEN_9997_1_C2_2017
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula, latex_fraction
+from blueprints.type_alias import DIMENSIONLESS
+from blueprints.validations import raise_if_negative
+
+
+class Form2Dot4DesignValueGeotechnicalParameter(Formula):
```

Comment:
We try to avoid plain float and int if possible. DIMENSIONLESS is a type alias of float (check type_alias.py), but there a lot of other type aliasses inside Blueprints (ie. MM2, HOURS, KN, etc). We thus try to use it to indicate the expected outcome of the calculation.

Reply:
Changed unit to N